rasa/core/processor.py

```diff
@@ -143,6 +175,7 @@ def predict_next(self, sender_id: Text) -> Optional[Dict[Text, Any]]:
         Optionally save the tracker if `should_save_tracker` is `True`. Tracker saving
         can be skipped if the tracker returned by this method is used for further
         processing and saved at a later stage.
+
```

Comment:
can you please update that in our python conventions? I always forget which we decided for :-D